frontend/kesaseteli/employer/src/components/application/steps/step1/Step1Employer.tsx

```diff
@@ -1,13 +1,15 @@
 import ApplicationForm from 'kesaseteli/employer/components/application/ApplicationForm';
 import ActionButtons from 'kesaseteli/employer/components/application/form/ActionButtons';
 import EmployerForm from 'kesaseteli/employer/components/application/steps/step1/EmployerForm';
+import useSetCurrentStep from 'kesaseteli/employer/hooks/application/useSetCurrentStep';
 import { useTranslation } from 'next-i18next';
 import React from 'react';
 
 const Step1Employer: React.FC = () => {
   const { t } = useTranslation();
+  useSetCurrentStep();
   return (
-    <ApplicationForm stepTitle={t('common:application.step1.header')}>
+    <ApplicationForm title={t('common:application.step1.header')}>
```

Comment:
nont important, but if for the step you have common parts for the string path, you can make a variable, like its done in benefit. looks a bit shorter then